source/assets/stylesheets/styles/_photo_gallery.scss

```diff
@@ -0,0 +1,49 @@
+.photo-gallery {
+  margin-top: 10px;
+  background-color: $white;
+  padding: 30px 0 30px 20px;
+
+  @include breakpoint($tablet-breakpoint) {
+    overflow: auto;
+    padding: 20px 0 0 20px;
+  }
+}
+
+.photo-gallery-container {
+  box-sizing: border-box;
+  padding-right: 20px;
+  margin-bottom: 40px;
+  width: 100%;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    width: 33.33%;
+  }
+
+  &:last-of-type {
+    margin-bottom: 0;
+  }
+}
+
+.photo-gallery__introduction {
+  @include responsive-text;
+  color: $purple;
+  font-weight: 300;
+  line-height: 1.6;
+  margin-bottom: 20px;
+  padding-right: 20px;
+
+  @include breakpoint($tablet-breakpoint) {
+    margin-bottom: 50px;
+    line-height: 1.3em;
+  }
+}
+
+.photo-gallery-image {
+  overflow: auto;
+  min-height: 225px;
+}
+
+.gallery__image {
```

Comment:
I'm not sure if this class fits neatly into the BEM system. I'm getting confused with the class in the rule above.

Could we nest an `img` selector inside the above rule and lose the class here?

Reply:
Yeah I spent a long time agonising over naming. I'll give your suggestions a go 😄